src/styles/breadcrumbs.css

```diff
@@ -1,3 +1,74 @@
 .breadcrumb-right-tag .breadcrumb-item:last-child  {
     max-width: none !important;
-}
\ No newline at end of file
+}
+
+.gus-breadcrumbs {
+    font-size: 1.5rem !important;
+    line-height:2.2rem !important;
+    color: #0b0c0c;
+    width: 100%;
+    height: 4.2rem;
+    position:relative;
+    margin-top: 10px;
+    margin-left: 5px;
+    margin-bottom: 0px;
+  }
+.gus-breadcrumbs a {
+    color:#0b0c0c;
+    text-decoration: none;
+}
+.gus-breadcrumbs a:hover {
+    color:#0b0c0c;
+    text-decoration: underline;
+}
+  @media (min-width: 40.0625em) {
```

Comment:
For consistency, I'd recommend using the Bootstrap min-width values to align with the existing responsive grid: https://getbootstrap.com/docs/5.3/layout/breakpoints/#min-width

Reply:
have changed this to 576px to match bootstrap